src/p5.dance.js

```diff
@@ -111,7 +110,11 @@ module.exports = class DanceParty {
   }
 
   addCues(timestamps) {
-    this.world.cues = timestamps;
+    // Sort and de-duplicate
+    const numSort = (a,b) => a - b;
```

Comment:
Also need to de-duplicate?

Reply:
Whoops! Leftover comment from when I thought we'd need to de-duplicate. The while loop in updateEvents means that duplicates are already handled as long as they are sorted. I'll update that comment.

Reply:
Don't we want to de-duplicate?  I guess it's fine for the `while` loop to take care of it... but sort of prefer to sanitize the data.